daemon/container_operations_windows.go

```diff
@@ -35,6 +43,62 @@ func detachMounted(path string) error {
 	return nil
 }
 
+func (daemon *Daemon) setupSecretDir(c *container.Container) (setupErr error) {
+	if len(c.SecretReferences) == 0 {
+		return nil
+	}
+
+	localMountPath := c.SecretMountPath()
```

Comment:
Not sure you need a `localMountPath` variable. You can pass `c;SecretMountPath()` in directly to the defer. eg

```
defer func (localMountPath string) {
...
} (c.SecretMountPath())